extern/nlohmann/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2022 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.14 )
+include( FetchContent )
+
+# Tell the user what's happening.
+message( STATUS "Building nlohmann as part of the Detray project" )
+
+# Declare where to get Thrust from.
+set( DETRAY_NLOHMANN_SOURCE
+   "GIT_REPOSITORY;git@github.com:nlohmann/json.git;GIT_TAG;v3.10.0"
```

Comment:
:thinking: I myself much more prefer to download TGZ files, as the repositories can be quite a bit bigger to download. So instead I'd suggest:

```
"URL;https://github.com/nlohmann/json/archive/refs/tags/v3.10.5.tar.gz;URL_MD5;5b946f7d892fa55eabec45e76a20286b"
```

(You can also use 3.10.0, but we may as well just use the latest one by default.) Which is actually what we do for most of our externals...

This was dropped from my "review" for some reason. :confused:

Reply:
alright, I saw that you did that similarly with the others, will update.